map/src/main/kotlin/edu/artic/map/MapFragment.kt

```diff
@@ -738,4 +926,34 @@ class MapFragment : BaseViewModelFragment<MapViewModel>() {
             tourArgument.onNext(tour)
         }
     }
-}
\ No newline at end of file
+}
+
+
+/**
+ * We only want to display [ArticObject] annotations that are within 15 meters
+ * of the center of the map.
+ *
+ * @param bounds the restrictions of
+ * [the map's viewport][com.google.android.gms.maps.Projection.getVisibleRegion]
+ */
+private fun LatLng.isCloseEnoughToCenter(bounds: LatLngBounds): Boolean {
+    return bounds.contains(this) && bounds.center.distanceTo(this) < 15
+}
+
+/**
+ * Alias to [Location.distanceBetween], where 'this' is the first param and 'other' is the second.
+ *
+ * @return a distance, in meters
+ */
+private fun LatLng.distanceTo(other: LatLng): Float {
```

Comment:
move this to `MapUtils` method. no need to pollute the `MapFragment` class with extra logic especially if we end up needing it somewhere else. :)

Reply:
Yeah, good call.